src/MetaMorpho.sol

```diff
@@ -284,6 +285,15 @@ contract MetaMorpho is ERC4626, ERC20Permit, Ownable2Step, Multicall, IMetaMorph
         }
     }
 
+    function disableMarket(Id id) external onlyCuratorRole {
```

Comment:
```suggestion
    function prepareMarketdisabling(Id id) external onlyCuratorRole {
```
`prepareMarketdisabling ` is ugly but I'm sure we can find better names for this. 

The fact is that function is just the first step for disabling a market.

Reply:
`submitDisableMarket` let's keep the classic name